website/source/docs/auth/oci.html.md

```diff
@@ -0,0 +1,209 @@
+    
+    `export VAULT_ADDR='[http://127.0.0.1:8200'](http://127.0.0.1:8200')`
+      
+1.  Enable the OCI Auth plugin.
+
+    * Create a file named authenable.json.
+
+    ```
+    {"config":{"audit_non_hmac_request_keys":null,"audit_non_hmac_response_keys":null,"default_lease_ttl":"0s","max_lease_ttl":"0s"},"description":"","local":false,"seal_wrap":false,"type":"oci"}
+    ```
+    
+    * Run the below command, replacing $roottoken with the root token you copied earlier.
+
+    ```     
+        curl --header "X-Vault-Token: $roottoken" --request POST \       
+        --data @authenable.json \        
+        http://[127.0.0.1:8200/v1/sys/auth/oci](127.0.0.1:8200/v1/sys/auth/oci)
+    ```
+
+1.  Create the vault admin policy.
+    * Determine what [set of policies](https://learn.hashicorp.com/vault/identity-access-management/iam-policies) best suits your requirements.
+    * Create a file named vaultadminpolicy.json. Configure the policy below according to your requirements.
+
+    ```       
+        {       
+        "policy": "path \"auth/oci/*\"       
+        {       
+        capabilities = [\"create\", \"read\", \"update\", \"delete\", \"list\"]        
+        }        
+        path \"sys/auth/*\"       
+        {       
+        capabilities = [\"read\", \"list\"]       
+        }       
+        path \"sys/policy\"       
+        {       
+        capabilities = [\"read\"]        
+        }
+        path \"sys/policy/*\"
+        {        
+        capabilities = [\"create\", \"read\", \"update\", \"delete\", \"list\"]        
+        }        
+        path \"secret/*\"        
+        {        
+        capabilities = [\"create\", \"read\", \"update\", \"delete\", \"list\"]        
+        }        
+        path \"sys/health\"        
+        {        
+        capabilities = [\"read\"]        
+        }"        
+        }
+    ```                          
+
+    * Run the command to create the policy.
+
+    ```        
+        curl \       
+        --header "X-Vault-Token: $roottoken" \       
+        --request PUT \       
+        --data @vaultadminpolicy.json \       
+        http://[127.0.0.1:8200/v1/sys/policy/vaultadminpolicy](127.0.0.1:8200/v1/sys/policy/vaultadminpolicy)
+    ```
+
+1.  Configure your home tenancy in the vault, so that only users or instances from your tenancy will be allowed to log into vault through the OCI Auth plugin.  
+    * Create a file named hometenancyid.json with the below content, using the tenancy OCID. To find your tenancy OCID, see [https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm).       
+
+        `{"configValue":"your tenancy ocid here"}`
+        
+    * Configure the homeTenancy parameter in the vault.
+       
+    ```     curl --header "X-Vault-Token: $roottoken" --request PUT \       
```

Comment:
Lets follow the formatting from other commands and drop the `curl` down to the line under the pre-formatting, aligned with the rest of the commands on other lines.

Reply:
Done